aws-amplify/docs
# Update csp

#### Description of changes:
- Remove `prefetch-src`

Staging site: https://update-csp.d3kzzkiugubm89.amplifyapp.com/

To test:
1. Open up the site
2. Open up dev tools to look at the `Console`
3. Verify that there isn't a console error for `Unrecognized Content-Security-Policy directive 'prefetch-src'.`
   - We have some other errors, but I think that's because it's on the amplifyapp.com domain and related to the adobe analytics stuff. It should be okay to ignore since it's just a staging site

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
